# AutoEncoder

Course Project for CMU 02710, 23 Spring

Author:     Li Chen

Email:      lic2@andrew.cmu.edu

## Description: 

The emergence of single-cell RNA sequencing (scRNA-seq) has been a groundbreaking development that has significantly improved our comprehension of cellular heterogeneity, allowing for gene expression profiling at the individual cell level. However, scRNA-seq data pose significant challenges for downstream analysis due to their high dimensionality, noise, and sparsity. Therefore, dimension reduction is a crucial step in scRNA-seq data analysis, as it seeks to reduce the high-dimensional gene expression data into a lower-dimensional representation while retaining the biological variation of interest. 

To accomplish this objective, various dimensionality reduction methods have been developed and applied in scRNA-seq data analysis. In our project, we have chosen a dataset and a set of metrics to evaluate different dimensionality reduction methods. Additionally, we aim to create code that is easily reproducible, modifiable, and useful for future benchmarking efforts.

I ran the notebook on Colab with standard GPU + standard RAM.

## Data

  Data is downloaded from Tabula Muris (https://tabula-muris.ds.czbiohub.org/).
